pkg/providers/vsphere/contentlibrary/content_library_utils.go

```diff
@@ -277,18 +304,26 @@ func populateImageStatusFromOVFDiskSection(imageStatus *vmopv1.VirtualMachineIma
 	imageStatus.Disks = make([]vmopv1.VirtualMachineImageDiskInfo, len(diskSection.Disks))
 
 	for i, disk := range diskSection.Disks {
-		diskDetail := vmopv1.VirtualMachineImageDiskInfo{}
+		name := disk.DiskID
+		if name == "" {
+			name = strconv.Itoa(i)
```

Comment:
Same comment here. 
Do you want to add a prefix like "disks-" to make this more readable?